pkg/version/base.go

```diff
@@ -26,6 +26,7 @@ var (
 	gitVersion   = "v0.0.0-master"
 	gitCommit    = "unknown" // sha1 from git, output of $(git rev-parse HEAD)
 	gitTreeState = "unknown" // state of git tree, either "clean" or "dirty"
+	gitRevision  = "unknown" // short sha1 for git, output of $(git rev-parse --short HEAD)
```

Comment:
```suggestion
	gitAbbreviativeCommit = "unknown" // short sha1 from git, output of $(git rev-parse --short HEAD)
```
Follow the concept `abbreviative` from git. Which also removes the ambiguity between `gitVersion` and `gitRevision`.
What do you think?

Reply:
Thanks @RainbowMango 
Got it
Change variable name done.